tests/e2e/test_rosacli_cluster.go

```diff
@@ -241,6 +242,47 @@ var _ = Describe("Edit cluster",
 
 			})
 
+		It("validation for create/delete upgrade policies for hypershift clusters via rosacli should work well - [id:73814]",
```

Comment:
As a precaution measure, I would suggest to add a `defer` which would remove the `upgrade`.
Just in case an upgrade step below would not report an error and the upgrade would get scheduled.
wdyt ?

Reply:
Done, Thanks